<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingForm</class>
 <widget class="QDialog" name="TrainingForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(139,233,159);
color: rgb(85, 13, 57);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Оберіть налаштування для тренування:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Обрати одну тему</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(251, 237, 217);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="emptyLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Обрати один білет</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="ticketLabel">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(251, 237, 217);</string>
             </property>
             <property name="text">
              <string>Номер білету:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(251, 237, 217);</string>
             </property>
             <property name="suffix">
              <string> білет</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="emptyLabel_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Час на тренування:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(251, 237, 217);</string>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="minimumTime">
          <time>
           <hour>0</hour>
           <minute>1</minute>
           <second>0</second>
          </time>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::MinuteSection</enum>
         </property>
         <property name="displayFormat">
          <string>mm:ss</string>
         </property>
         <property name="time">
          <time>
           <hour>0</hour>
           <minute>1</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="styleSheet">
        <string notr="true">QPushButton:hover{
	border-width: 0px;
	border-radius: 20%;
}
QPushButton {
	border: 4px solid;
	border-color: rgb(52, 101, 164);
	border-radius: 6px;
	background-color: rgb(189, 214, 254);
	color: rgb(9, 4, 32);
	padding: 10px;
	outline: none;
}

QPushButton:flat {
	border: none; 
}

QPushButton:default {
border-color: rgb(92, 53, 102);
}

QPushButton:disabled{
background-color: rgb(211, 215, 207);
}</string>
       </property>
       <property name="text">
        <string>До головного меню</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="emptyLabel_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover{
	border-width: 0px;
	border-radius: 20%;
}
QPushButton {
	border: 4px solid;
	border-color: rgb(52, 101, 164);
	border-radius: 6px;
	background-color: rgb(189, 214, 254);
	color: rgb(9, 4, 32);
	padding: 10px;
	outline: none;
}

QPushButton:flat {
	border: none; 
}

QPushButton:default {
border-color: rgb(92, 53, 102);
}

QPushButton:disabled{
background-color: rgb(211, 215, 207);
}</string>
       </property>
       <property name="text">
        <string>Розпочати тренування</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>TrainingForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>TrainingForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
